parcels/particlesets/collections.py

```diff
@@ -619,7 +619,7 @@ def clear(self):
         pass
 
 
-class ParticleCollection(ABC, Collection):
+class ParticleCollection(Collection):
```

Comment:
why leaving out ABC ? ParticleCollection is still itself also an abstract class - and Python >=3.5 does actually facilitate multi-class inheritance (one of the actually cool features of newer Python versions).

Reply:
It does facilitate multi-class inheritance, but not in this way. `Collection` already inherits from `ABC`. If `ParticleCollection` inherits from both, Python will not know what to do: `Cannot create a consistent method resolution order (MRO) for bases ABC, Collection`.
Besides, `ParticleCollection` does not explicitly need to inherit from `ABC` anyway. As long as not all abstract methods are instantiated, any subclass of Collection will be abstract itself. So leaving `ABC` out here has no negative consequences.